src/index.js

```diff
@@ -0,0 +1,13 @@
+import './main.css';
+
+const scores = document.querySelector('.scores');
+
+scores.innerHTML = `
+<ul>
+<li>Name:100</li>
+<li class="li-grey">Name:10</li>
+<li>Name:200</li>
+<li class="li-grey">Name:40</li>
+<li>Name:60</li>
+<li class="li-grey">Name:60</li>
+</ul>`;
```

Comment:
- [ ] good job on making the project look as required 👍 however one of the requirements of this project is that you use ES6 modules, with import and export. which means you need to use arrow functions. below you can find a small example of how this should look like 
```
const index = () => {
  your code...
};

export default index();
```